packages/core/src/routes/profile/index.ts

```diff
@@ -22,6 +24,41 @@ export default function profileRoutes<T extends UserRouter>(
     return;
   }
 
+  router.patch(
+    '/profile',
+    koaGuard({
+      body: z.object({
+        name: z.string().nullable().optional(),
+        avatar: z.string().url().nullable().optional(),
+      }),
+      response: z.object({
+        name: z.string().nullable().optional(),
+        avatar: z.string().nullable().optional(),
+      }),
```

Comment:
should we return the full user profile object for a PATCH API?

Reply:
It is hard to the the full profile object, because we should use scopes to filter fields. We encourage the user to use `/oidc/me` api.